app/assets/stylesheets/image_x.scss

```diff
@@ -12,6 +12,12 @@ $image-thumb-slider-border-color: #000;
   position: relative;
 }
 
+##{$namespace}-image-x {
+  height: 200%;
+  width:100%;
```

Comment:
Properties should be ordered height, position, width<br>Colon after property should be followed by one space